guarddog/analyzer/sourcecode/shady-links.yml

```diff
@@ -25,6 +25,7 @@ rules:
         - pattern-either:
             # complete domains
             - pattern-regex: (http[s]?:\/\/[^\n\[\/\?#"']*?(bit\.ly|discord\.com|workers\.dev|transfer\.sh|filetransfer\.io|sendspace\.com|appdomain\.cloud|backblazeb2\.com\|paste\.ee|ngrok\.io|termbin\.com|localhost\.run|webhook\.site|oastify\.com|burpcollaborator\.me)\/)
+            - pattern-regex: (http[s]?:\/\/[^\n\[\/\?#"']*?(oast\.(pro|live|site|online|fun|me)|api\.telegram\.org|rentry\.co)\/)
```

Comment:
IIRC we had api.telegram.org in the past and it triggered quite a lot of noise

Reply:
ok, will run it against the top 1k before merge

Reply:
I just had 1 from in the top 1 pypi and 0 hits in top 1k npm.
(could not find it in the repo history)
I'll proceed